ash/Cargo.toml

```diff
@@ -14,6 +14,7 @@ rust-version = "1.59.0"
 [dependencies]
 libloading = { version = "0.7", optional = true }
 
+# note: features are also documented at the top of lib.rs
```

Comment:
I'm not too fussed about this one, perhaps we might just remove the docs here entirely as they're not easily visible at all, and I've not seen many crates do this.

Reply:
Sounds good. No harm in leaving the original comments there though, some people might look in Cargo.toml for feature info.